flow/Net2.actor.cpp

```diff
@@ -354,7 +358,12 @@ class BindPromise {
 			if (error) {
 				// Log the error...
 				{
-					TraceEvent evt(SevWarn, errContext, errID);
+					std::optional<TraceEvent> traceEvent;
```

Comment:
What if we just declare default initialized TraceEvent here? At the end std::optional is doing also the same internally

Reply:
As shown below, some, not all, of the TraceEvents to be created are auditable and therefore require a different constructor. It is also made intentionally impossible to promote the default-constructed TraceEvent to auditable and therefore non-suppressible.